src/interface.jl

```diff
@@ -417,7 +409,9 @@ end
 
 # Fallback to a MethodError to avoid stack overflow
 @doc "$(_doc_exp_id)"
-function ManifoldsBase.exp!(G::LieGroup, h, e::Identity, X, t::Number=1)
+function ManifoldsBase.exp!(
+    G::LieGroup{𝔽,Op}, h, e::Identity{Op}, X, t::Number=1
+) where {𝔽,Op<:AbstractGroupOperation}
```

Comment:
I think this was correct before: it's supposed to throw an error when group operations don't match.

Reply:
Ah formerly it was just a fallback, but this here seems ring since Op should not be the same necessarily? Will check.

Reply:
Hm, the only thing I can manage when I try to change this is to increase the number of ambiuities. If you see how to make this more general, feel free to do so.

Reply:
Those ambiguities look real, that is `exp!` for `PowerLieGroup` likely won't work if the base point is `Identity`. I'm not sure if you'd like to make such test through the new framework or keep it separate?

Reply:
Currently there are non if we leave the code here as is; I will check with power – and sure through the framework, that exp works with identity.

Reply:
If I see that correctly the test with identity should already be running and working. cf https://github.com/JuliaManifolds/LieGroups.jl/blob/d16f091928f941f30daecbf419bc6551aec7334f/src/groups/power_group.jl#L204-L221, which does have coverage.

Reply:
Hm, I see, though I'm not sure if this modified method actually can handle its original purpose. Anyway, maybe let's care about it when it actually happens.